nengo_loihi/tests/test_learning.py

```diff
@@ -208,7 +208,7 @@ def test_multiple_pes(init_function, allclose, plt, seed, Simulator):
 
         probe = nengo.Probe(output, synapse=0.1)
 
-    simtime = 0.5
+    simtime = 0.6
```

Comment:
Why?

Reply:
This test was failing, but I'm pretty sure it was just because the seed changed (running it various times with different seeds it would sometimes randomly fail).  Increasing the time made it pass reliably.

Reply:
If switching to 0.6 makes it work for a wider range of seeds, probably worth the change, then. Maybe we can add a short statement in the commit message saying this?

As I mentioned below, the randomness of this test is also fixed in #140, so between the two changes, should be solid!